libraries/ms-drivers/src/x86/pca9539r_gpio_expander.c

```diff
@@ -1,5 +1,5 @@
 #include "pca9539r_gpio_expander.h"
-
+#include "status.h"
```

Comment:
Nitpick: the "include order" item in our [coding standards](https://uwmidsun.atlassian.net/wiki/spaces/ELEC/pages/9306218/Coding+Standards) dictates that there should be a blank line between the #include of this C file's header file (here `#include "pca9539r_gpio_expander.h"`) and the rest of the includes, and there should be a blank line between the last include and the rest of the file.